packages/shared/src/utils/currency/currency.ts

```diff
@@ -36,21 +58,27 @@ export const reorderCurrencies = (list, type = 'fiat') => {
 
 export const AMOUNT_MAX_LENGTH = 10;
 
-export const getRoundedNumber = (number, currency) => {
+export const getRoundedNumber = (number: number, currency: string) => {
     return Number(Number(number).toFixed(getDecimalPlaces(currency)));
 };
 
-export const getFormattedText = (number, currency) => {
+export const getFormattedText = (number: number, currency: string) => {
     return `${addComma(number, getDecimalPlaces(currency), isCryptocurrency(currency))} ${currency}`;
 };
 
-export const formatMoney = (currency_value, amount, exclude_currency, decimals = 0, minimumFractionDigits = 0) => {
-    let money = amount;
+export const formatMoney = (
+    currency_value: string,
+    amount: number | string,
+    exclude_currency?: number,
```

Comment:
@niloo-fs according to the way we use `formatMoney()` component across the app, we pass a `boolean` value to it as the 3rd parameter all the time. So the type of exclude_currency here is what's causing circleci tests fail on your PR:
```suggestion
    exclude_currency?: boolean || number,
```
also, we pass a `number` to it as a third param in the test file only - [currency.spec.ts](packages/shared/src/utils/currency/__tests__/currency.spec.ts). so to me, it makes more sense to fix the test by passing `true` instead of `1` at line 45. 🙂
then it would be:
```suggestion
    exclude_currency?: boolean,
```
and tests would not fail😉